examples/counter/common/src/commonMain/kotlin/org/reduxkotlin/examples/counter/Reducer.kt

```diff
@@ -0,0 +1,15 @@
+package org.reduxkotlin.examples.counter
+
+import org.reduxkotlin.Reducer
+
+
+class Increment
+class Decrement
+
+val reducer: Reducer<Int> = {state, action ->
+    when (action) {
+        is Increment -> state + 1
```

Comment:
A sealed reducer could eliminate the else statement

Reply:
True..I'd like to keep this a minimal example.  I see sealed classes as an optional, more advanced usage.  I would not point people to this for a "best practices" example